[FEATURE] Add a Core Mergers Section to the "Howtos" (#203)
TYPO3-Documentation/TYPO3CMS-Guide-ContributionWorkflow: Documentation/HandlingAPatch/Review.rst

#. Now press :guilabel:`Reply`.

   Using the :guilabel:`Reply` button, you can post your comments
   (and optionally add an additional note).

   .. figure:: /Images/External/Gerrit/CoreMergers/VoteUser.png
      :class: with-shadow

      Vote by clicking the :guilabel:`Reply` button

   Of course you should also :ref:`vote <gerrit-voting>` for the change

>    (Be graceful with -1 votes though).

Reviewer: ```suggestion
   (Be discerning with -1 votes though).
```
I'm not sure what graceful meant. I'm not sure that discerning is much better here. I guess we're saying, use it sparingly? or are we saying be mindful - know what you're doing? It's not clear to me. but maybe it's clear in context, so just ignore these comments if you want.

Author: I think, programmers like using the word gracefull in that context

Vote
====

In order to comment or vote on a change you can click on the :guilabel:`Reply`
and enter your comment. Here, you can also apply your votes.

.. figure:: /Images/External/Gerrit/CoreMergers/VoteUser.png
   :class: with-shadow

   Chose your vote, say something nice and click :guilabel:`SEND`

*  :guilabel:`+1` : you approve of the patch

> *  :guilabel:`-1` : you do not approve, in this case give some reason as a comment

Reviewer: ```suggestion
*  :guilabel:`-1` : you do not approve, in this case give your reason as a comment
```

Votes from the Bamboo build server (user *TYPO3com*) do not count. This means
that a patch which is fully reviewed usually has at least 3 **Verified** :guilabel:`+1`
votes, two from humans and one from Bamboo.

**Authors should not vote for their own patches**, unless the patch has been changed
substantially by other developers.

As soon as the patch has reached the approved status by getting a :guilabel:`+2` on
**Code Review** and **Verified**, a Core Merger can decide to push the
:guilabel:`Submit` button, finally pushing it to the main repository.

Hints on voting -1
------------------

See also :ref:`reviewPatch` about the policies on voting and how to vote.

In general, :guilabel:`-1` on reading and/or testing of a patch is a mechanism used to
improve a patch. Still, -1 still takes a risk to kill someone elses patch
and it usually actively prevents a merge. There are ways to override a -1, but
those are not pushed through in real live Gerrit habits. In general, if voting
-1, you take some responsibility for this patch by saying "This one shouldn't
be solved until this or that is fixed". Some hints on using -1 in reviews:

> *  Think about your vote and always give a sane explanation. "-1 looks ugly" is

Reviewer: ```suggestion
*  Think about your vote and always give a logical explanation. "-1 looks ugly" is
```
OR: "meaningful" or "thoughtful".

   not enough.
*  If a patch is broken, does not fix the issue, is bogus, architecturally wrong
   or collides with other goals, a -1 is clearly ok.
*  -1 can also be used if you are actively working on a patch and want to prevent
   a quick merge: "-1, working on it now, will push soonish".

> *  -1 may be ok if you have general doubts but you can not pin point it and need
>    a second opinion: "Hey, this solution looks somehow weird and I doubt this is
>    what we should do here. I think we should have a statement by x or y who have
>    a deeper knowledge of this subsystem to have an eye on that. I do not want
>    this patch to be merged until this was sorted out and will vote -1 for now
>    for this reason."

Reviewer: ```suggestion
*  -1 may be ok if you have general doubts but you can not pinpoint it and need
   a second opinion: "Hey, this solution looks somehow weird and I doubt this is
   what we should do here. I think we should ask person x or person y, who has
   a deeper knowledge of this subsystem, to take a look at it. I do not want
   this patch to be merged until this is sorted out and will vote -1 for now
   for this reason."
```
